lms/djangoapps/shoppingcart/models.py

```diff
@@ -1962,34 +1965,48 @@ def purchased_callback(self):
         self.course_enrollment.send_signal(EnrollStatusChange.upgrade_complete,
                                            cost=self.unit_cost, currency=self.currency)
 
+    def get_how_to_unenroll_url(self):
+        user_preferences = get_user_preferences(self.order.user)
+        pref_language = user_preferences.get('pref-lang')
+        support_url = build_help_center_url(pref_language)
+        if 'hc/' not in support_url:
+            support_url += 'hc/en-us'
+
+        # please check prod-1418 for more detail.
+        how_to_unenroll_link = '{support_link}/articles/115015783627'.format(support_link=support_url)
```

Comment:
I think we should not hard code this article number here because it's only edX specific. We should move it into a variable in internal vars.

Reply:
I think we should put the hardcode url in internal var. Because that url is only valid for edx and if we get the support link dynamically it would't not work for open edx. So i think we should either hard code the url or remove it for open edx.

Reply:
In that case we shouldn't add the link into instructions at all.